bosshud/ze_lost_terminus_b10.txt

```diff
@@ -0,0 +1,43 @@
+"math_counter"
+{
+    "0"
+    {
+        "HP_counter"        "1bosshpc"
+        "HPbar_counter"     "1bosshp2"
+        "HPbar_max"         "20"
+        "HPbar_min"         "0"
+        "HPbar_default"     "20"
+        "HPbar_mode"        "2"
+        "CustomText"        "Boss"
```

Comment:
CustomText is optional, there is no reason to provide it if you don't have a custom boss name. This goes for the other entries below as well.